docs/client.rst

```diff
@@ -1,19 +1,19 @@
-Client
-======
+DockerClient
```

Comment:
I think this should just be "Client" -- it's the title of the documentation page about clients, not the specific object name.

In the same way below, it says "Creating a client", not "Creating a DockerClient".